dts/x86/yaml/arduino101.yaml

```diff
@@ -0,0 +1,42 @@
+---
+title: arduino101 Uart
```

Comment:
Is the uart really arduino101 specific, isn't the UART QMSI?  This should probably be "Intel QMSI Uart" (fixup other uart references accordingly)